gui/design.lua

```diff
@@ -120,8 +120,42 @@ function DimensionsOverlay:preUpdateLayout(parent_rect)
     self.frame.h = parent_rect.height
 end
 
+---
+--- RightClickOverlay
+---
+
+RightClickOverlay = defclass(RightClickOverlay, overlay.OverlayWidget)
+RightClickOverlay.ATTRS{
+    desc='When drawing boxes, makes right click cancel selection instead of exiting.',
+    default_enabled=true,
+    viewscreens={
+        'dwarfmode/Designate',
+        'dwarfmode/Burrow/Paint',
+        'dwarfmode/Stockpile/Paint',
+        'dwarfmode/Zone/Paint',
```

Comment:
Ha - I need to handle this case for DimensionsOverlay too. Thanks!

Reply:
after you made the edit, did you run `:lua require('plugins.overlay').rescan()`? changes to overlay widgets need to be reloaded at the framework level.